packages/@webex/plugin-meetings/test/unit/spec/annotation/annotation.ts

```diff
@@ -0,0 +1,349 @@
+import MockWebex from '@webex/test-helper-mock-webex';
```

Comment:
issue: The convention is to have the test file names the same as file name they are testing. This file should be called index.ts

Reply:
Done